src/Kdyby/Doctrine/Diagnostics/Panel.php

```diff
@@ -484,12 +458,6 @@ public static function renderException($e, Nette\DI\Container $dic)
 				];
 			}
 
-		} elseif ($e instanceof Kdyby\Doctrine\DBALException && $e->query) {
```

Comment:
PDOException is always wrapped, so there is no need to render the query also at it's level